commit/merkleroot/outcome.go

```diff
@@ -363,6 +363,9 @@ func checkForReportTransmission(
 		OutcomeType:                     ReportInFlight,
 		OffRampNextSeqNums:              previousOutcome.OffRampNextSeqNums,
 		ReportTransmissionCheckAttempts: previousOutcome.ReportTransmissionCheckAttempts + 1,
+		// Carry over the previous roots since they're still in-flight.
+		// We won't re-report since outcome type is ReportInFlight.
+		RootsToReport: previousOutcome.RootsToReport,
```

Comment:
may as well group this variable & comment /w `OffRampNextSeqNums` since they're both carried over for the same reason

Reply:
@dimkouv can you share your thoughts? I don't really see advantages / disadvantages either way.